src/expandable-block/block.scss

```diff
@@ -167,4 +167,35 @@
       margin-left: $iui-s;
     }
   }
+
+  &.iui-borderless {
+    @include themed {
```

Comment:
I think I made a mistake here. If the background color and border are removed in iui-header and iui-expandable-content, this may be redundant